src/mbgl/renderer/debug_bucket.cpp

```diff
@@ -9,9 +10,20 @@
 
 using namespace mbgl;
 
-DebugBucket::DebugBucket(const TileID id, const TileData::State state_) : state(state_) {
+DebugBucket::DebugBucket(const TileID id, const TileData::State state_, time_t modified_, time_t expires_)
+    : state(state_),
+      modified(modified_),
+      expires(expires_) {
     const std::string text = std::string(id) + " - " + TileData::StateToString(state);
     fontBuffer.addText(text.c_str(), 50, 200, 5);
+
+    if (modified > 0 && expires > 0) {
+        const std::string modifiedText = "modified: " + util::rfc1123(modified);
```

Comment:
Can we find a shorter representation of the date, e.g. "Y-m-d H:i:s" and imply GMT?